cmake/GoogleUtilities.cmake

```diff
@@ -34,6 +34,12 @@ firebase_ios_add_framework(
   ${headers} ${sources}
 )
 
+# GoogleUtilities
+target_include_directories(
+  GoogleUtilities PRIVATE
+  $<BUILD_INTERFACE:${FIREBASE_EXTERNAL_SOURCE_DIR}/GoogleUtilities>
```

Comment:
`PRIVATE` properties only apply at build time anyway, so using `$<BUILD_INTERFACE:...>` is redundant.